docs/docs/integrations/api.md

```diff
@@ -295,3 +295,19 @@ Get ffprobe output for camera feed paths.
 ### `GET /api/<camera_name>/ptz/info`
 
 Get PTZ info for the camera.
+
+### `POST /api/events/<camera_name>/<label>/create`
+
+Create a manual API with a given `label` (ex: doorbell press) to capture a specific event besides an object being detected.
+
+```json
+{
+  "subLabel": "some_string", // add sub label to event
+  "duration": 30, // predetermined length of event (default: 30 seconds) or can be to null for indeterminate length event
+  "include_recording": true // whether the event should save recordings along with the snapshot that is taken
+}
+```
+
+### `POST /api/events/<event_id>/end`
```

Comment:
I think it makes sense for this to be a PUT since it's an update to an existing event.